runtime_builders/template_builder.py

```diff
@@ -48,10 +48,16 @@ def _resolve_and_publish(config_file, bucket):
     try:
         gcs_paths = []
         with open(config_file, 'r') as f:
-            project_cfg = json.load(f)
+            if 'json' in config_file:
```

Comment:
Not really, I just figured it didn't require much work to support both, to give people the option in case they really felt strongly about one or the other.